packages/ui/src/components/DatePicker/PreviousDateRangePicker.tsx

```diff
@@ -149,6 +250,56 @@ const PreviousDateRangePickerImpl = ({
 		}
 	}
 
+	const handleTimeChange = (
+		event: React.ChangeEvent<HTMLInputElement>,
+		input: 'start' | 'end',
+	) => {
+		const value = event.target.value
+		const isValid = [
+			TIME_INPUT_FORMAT,
+			TIME_INPUT_FORMAT_NO_SPACE,
+			TIME_INPUT_FORMAT_12_HOUR,
+			TIME_INPUT_FORMAT_12_HOUR_NO_SPACE,
+		]
```

Comment:
could this also support `1pm` or `1 pm` without a minute input, assuming that minutes are 00?

Reply:
I've just made an improvement to the time input format. Now, users can also input hours without minutes.

Reply:
nice! this works well now